### PR TITLE
Created note listing page UI

### LINKED ISSUE #1: 1. Create note listing page UI

### PR DESCRIPTION
**Pull request by cjjoel**
Fixes #1

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     parallelism: 1
     docker:
       - image: 'cypress/included:3.4.0'
-    working_directory: ~/neetoUIChallenge
+    working_directory: ~/neeto-ui-challenge
```
Comment (yedhink) [inline .circleci/config.yml:14]: Not sure why this naming change was made and also why this change was pushed in a PR named `Created note listing page UI`. It's a wrong and hacky practice. PRs should be atomic.

Do let me know your reasons.

File: app/javascript/src/components/Common/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  return (
+    <NeetoUIHeader
+      title={title}
+      menuBarToggle={() => setShowMenu(showMenu => !showMenu)}
+      actionBlock={
+        <>
+          <Input
+            autoComplete="off"
+            placeholder="Search Name, Email, Phone Number, etc."
+            prefix={<Search />}
```
Comment (yedhink) [inline app/javascript/src/components/Common/Header.jsx:17]:
```suggestion
prefix={Search}
```

File: app/javascript/src/components/Common/MenuBar.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        label={label}
+        count={count}
+        active={isActive && idx === 0}
+      />
+    ));
+
+  return (
+    <NeetoUIMenuBar showMenu={showMenu} title={title}>
+      {createBlocks(primaryBlocks, true)}
+
```
Comment (yedhink) [inline app/javascript/src/components/Common/MenuBar.jsx:27]: Please don't add unnecessary blank lines within JSX

File: app/javascript/src/components/Common/MenuBar.jsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        key={idx}
+        label={label}
+        count={count}
+        active={isActive && idx === 0}
+      />
+    ));
+
+  return (
+    <NeetoUIMenuBar showMenu={showMenu} title={title}>
+      {createBlocks(primaryBlocks, true)}
```
Comment (yedhink) [inline app/javascript/src/components/Common/MenuBar.jsx:26]: Why is this indented like this?

**Review by yedhink — COMMENTED** [review]
@cjjoel 

Few comments added. Go through them

File: .circleci/config.yml (hunk 1 above)
Comment (cjjoel) [reply .circleci/config.yml:14]: Sorry, It was supposed to be in the PR for setup. Since I have already merged that branch and it's only a small change I thought it wasn't a problem to include it in the branch I was working on.

File: app/javascript/src/components/Common/Header.jsx (hunk 2 above)
Comment (cjjoel) [reply app/javascript/src/components/Common/Header.jsx:17]: I get a warning(attached below) and Search icon won't be displayed if I do that. I think prefix prop expects JSX 
prefix prop error

File: app/javascript/src/components/Common/MenuBar.jsx (hunk 3 above)
Comment (cjjoel) [reply app/javascript/src/components/Common/MenuBar.jsx:27]: Removed it now.

File: app/javascript/src/components/Common/MenuBar.jsx (hunk 4 above)
Comment (cjjoel) [reply app/javascript/src/components/Common/MenuBar.jsx:26]: Are you talking about `{createBlocks(primaryBlocks, true)}`?

File: app/javascript/src/components/Common/Header.jsx (hunk 2 above)
Comment (yedhink) [reply app/javascript/src/components/Common/Header.jsx:17]: That's weird. Let's worry about this later then.

File: app/javascript/src/components/Common/MenuBar.jsx (hunk 4 above)
Comment (yedhink) [reply app/javascript/src/components/Common/MenuBar.jsx:26]: Ignore this comment. Thought NeetoUIMenuBar was a inline closing tag.
